README.md

```diff
@@ -233,11 +233,12 @@ OPTIONS:
     -f, --format <format>                      Output format of final status report (compact, detailed, json, markdown)
                                                [default: compact]
         --github-token <github-token>          GitHub API token to use when checking github.com links, to avoid rate
-                                               limiting [env: GITHUB_TOKEN=]
+                                               limiting [env: GITHUB_TOKEN=39cc0d59a69f0ac929174b52b3517216704fd977]
```

Comment:
not a good idea to expose the token

Reply:
Ouch. I revoked the token and made sure that it wasn't used in the meantime according to the Github logs. Thanks for letting me know.

Reply:
Also added 
```
#[structopt(long, env = "GITHUB_TOKEN", hide_env_values = true)]
```
to make sure this won't happen again.